### PR TITLE
Automatically embed youtube clips

### PR DESCRIPTION
**Pull request by spajus**
Borrowed the implementation from https://github.com/dejan/auto_html/blob/master/lib/auto_html/filters/youtube.rb

@ajanauskas @redross

### CODE REVIEW

**Comment by spajus** [comment]
Ping @vinted/backend

File: test/test_rails_autolink.rb [hunk 1]
```diff
@@ -162,6 +162,11 @@ def test_auto_link_other_protocols
     assert_equal generate_result(view_source), auto_link(view_source)
   end
 
+  def test_auto_link_youtube
+    link_raw = 'https://www.youtube.com/watch?v=mnl7Q_IBXbw'
+    assert_equal %{<div class="video youtube"><iframe width="500" height="281" src="//www.youtube.com/embed/mnl7Q_IBXbw" frameborder="0" allowfullscreen></iframe></div>}, auto_link(link_raw)
```
Comment (redross) [inline test/test_rails_autolink.rb:167]: Just wondering if it will be responsive?

Comment (spajus) [inline test/test_rails_autolink.rb:167]: I guess not, we would probably need to do some javascript magic for that.

**Comment by ajanauskas** [comment]
I think it should have some kind of option whether to embed youtube links. I believe we use auto links in api too and have no idea what would happen in mobile apps if this tag would be present

**Comment by spajus** [comment]
> I believe we use auto links in api too and have no idea what would happen in mobile apps if this tag would be present

Really? I thought this patched up version of auto_links was meant to be used in our web only, since the code that uses it uses webapp urls a lot.

**Comment by redross** [comment]
This does seem a bit outdated :) Close ?

**Comment by redross** [comment]
cc @tomasv

**Comment by spajus** [comment]
Who are these people? :smile: 
Oh wait, don't merge yet, I will add a backdoor first :smile:

**Comment by agejevasv** [comment]
Long time no see :beers:

**Comment by tomasv** [comment]
@spajus close your issue, because I can't. :smile:

**Comment by spajus** [comment]
Closing, owners too scared to merge :smile:

**Comment by laurynas** [comment]
Oh what a gang!! :) Greetings from autobahn near Dusseldorf! :de: 

imag0965

**Comment by ajanauskas** [comment]
lol

**Pull request closed without merging**